test/extensions/filters/common/expr/context_test.cc

```diff
@@ -2,9 +2,10 @@
 
 #include "extensions/filters/common/expr/context.h"
 
+#include "test/mocks/http/mocks.h"
```

Comment:
Why http mocks here? (and possibly elsewhere?)

Reply:
This test used `Http::TestRequestHeaderMapImpl` from upstream/mocks.h before.
I thought it's included from http mocks. But actually they are defined in test_common/utility.h. Now it's been fixed